### PR TITLE
Build and Deploy to Playground Demo (Netlify)

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/camunda/form-playground/issues/16

This one adds two GitHub actions to create and clean up `form-playground` demos with a form-js branch attached. Some things were mirrored from [`infra-global-github-actions`](https://github.com/camunda/infra-global-github-actions/tree/main/preview-env).

image

- Example: [preview creation](https://github.com/bpmn-io/form-js/actions/runs/4204150254/jobs/7294433364)
- Example: [clean up](https://github.com/bpmn-io/form-js/actions/runs/4204142528/jobs/7294415851)
=> [Resulting demo](https://demo-deploy-playground-demo--camunda-form-playground.netlify.app/)

Todo:
* [ ] Update [HTO Handbook](https://github.com/camunda/team-hto)

### CODE REVIEW

File: .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          repository_name: form-playground
+          new_branch_name: ${{ env.PREVIEW_BRANCH_NAME }}
+          access_token: ${{ secrets.ADD_TO_HTO_PROJECT_PAT }}
+      - name: Create GitHub deployment
+        uses: bobheadxi/deployments@88ce5600046c82542f8246ac287d0a53c461bca3
+        id: deployment
+        with:
+          step: start
+          env: ${{ env.PREVIEW_BRANCH_NAME }}
+          token: "${{ secrets.ADD_TO_HTO_PROJECT_PAT }}"
```
Comment (Langleu) [inline .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml:30]: you don't need to pass in a token.
GitHub will automatically supply you with a token on workflow start.
This token has some basic permissions in regards to your repository.

See [docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) for more info.

tl;dr
Can be replaced with: ```${{ secrets.GITHUB_TOKEN }}```

File: .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: DEPLOY_PLAYGROUND_PREVIEW
+on:
+  pull_request:
+    types: [ labeled, synchronize ]
+jobs:
+  deploy-preview:
+    # check whether the labeled event was deploy-preview || check whether on new commit of PR the label deploy-preview exists
+    if: github.event.label.name == 'deploy-preview' || contains( github.event.pull_request.labels.*.name, 'deploy-preview')
+    runs-on: ubuntu-20.04
```
Comment (maxdanilov) [inline .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml:9]: [suggestion] it's better to use a latest LTS (22.04) or even `ubuntu-latest` here.

Comment (pinussilvestrus) [reply .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml]: Thanks for the hint! I remember we pinned the version due to some difficulties with firefox (cf. https://github.com/bpmn-io/form-js/pull/384). But this is not a problem in these actions anyway.

File: .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: DEPLOY_PLAYGROUND_PREVIEW
```
Comment (vsgoulart) [inline .github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml:1]:
```suggestion
name: Deploy playground preview
```

File: .github/workflows/TEARDOWN_PLAYGROUND_PREVIEW.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+name: TEARDOWN_PLAYGROUND_PREVIEW
```
Comment (vsgoulart) [inline .github/workflows/TEARDOWN_PLAYGROUND_PREVIEW.yml:1]:
```suggestion
name: Teardown playground preview
```

**Review by vsgoulart — APPROVED** [review]
